crates/reader/src/workspace.rs

```diff
@@ -28,14 +28,13 @@ pub fn workspace_dir() -> String {
 }
 
 pub fn target_dir() -> String {
-    const JSON_KEY: &str = r#""target_directory":"#;
+    const JSON_KEY: &str = r#""target_directory":""#;
```

Comment:
Is this valid? This is performing a search for `"target_directory":""` which will fail right?

Reply:
I think the raw string literal is enclosed by `r#"` and `"#` so the resulting string is merely `"target_directory":"` instead of `"target_directory":`.

Reply:
But let's be consistent with `workspace_dir`. Should probably collapse the two into a single function.

Reply:
Fair point, I hadn't even spotted `fn workspace_dir()` :P